examples/studio/chat_complete.py

```diff
@@ -0,0 +1,28 @@
+from ai21 import AI21Client
+from ai21.models import RoleType
+from ai21.models.chat import ChatMessage
+
+system = "You're a support engineer in a SaaS company"
```

Comment:
Shouldn't this file be nested under chat/completions?

Reply:
I put it under examples/chat/chat_completions.py